recipes/caelus/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "caelus" %}
```

Comment:
It is a bit confusing that the package on PyPI is a different one. Do you also distribute the package there (with a different name)?

Reply:
We are not distributing the package on PyPI. Do you want me to use a different package name?

Reply:
I took a look at the PyPI package with the same name and it seems like a non-used and inactive project so I think in this special case it's fine to keep the name as-is.